src/components/IssueListItem.js

```diff
@@ -1,22 +1,30 @@
 import React from "react";
 import {FloatRight, FloatLeft, FlexColumn, FlexHeader, FlexCenter} from "../styles/Grid";
 import {chevronRight} from "../icons";
+import moment from "moment";
+import Octicon, {getIconByName} from "@primer/octicons-react";
 
-function IssueListItem({title, labels}) {
+function IssueListItem({title, labels, author, opened, type}) {
   return (
     <FlexHeader>
       <FloatLeft>
         <FlexCenter>
           <FlexColumn className="details">
-            <p>{title}</p>
-            <div>
+            <p>
+              <span style={{margin: 0}}>
+                <Octicon verticalAlign="middle" icon={getIconByName("issue-opened")} />
+              </span>
+              {title}
               {labels.data.length > 0 && labels.data.map(label => {
                 <span
                   style={{backgroundColor: `#${label.node.color}`}}
                   key={label.node.id}>
                   {label.node.name}
-                </span>;
+                </span>
               })}
```

Comment:
The labels are not rendering because the `{` after the map needs to be a `(`.

The `{` needs to include a `return` after it, but most React devs just use the `(` because it infers the `return` statement.